docs/source/contributing/tests.rst

```diff
@@ -109,21 +105,21 @@ All the tests are failing
 -------------------------
 
 Make sure you have completed all the steps in :ref:`contributing/setup` successfully, and
-can launch ``jupyterhub`` from the terminal.
+can launch ``jupyterhub`` from the terminal as well as the web browser.
```

Comment:
As before, do you mean opening JupyterHub in a browser after starting it from a terminal?